modules/javafx.web/src/main/java/com/sun/webkit/network/HTTP2Loader.java

```diff
@@ -576,8 +563,6 @@ private Void didFail(final Throwable th) {
                 throw th;
             } catch (MalformedURLException ex) {
                 errorCode = LoadListenerClient.MALFORMED_URL;
-            } catch (@SuppressWarnings("removal") AccessControlException ex) {
-                errorCode = LoadListenerClient.PERMISSION_DENIED;
```

Comment:
Excellent. This was on my list for [JDK-8342998](https://bugs.openjdk.org/browse/JDK-8342998), so one less thing for me to do for that (actually, two less things, since you also took care of the one in `URLLoader.java`).